main.nf

```diff
@@ -118,18 +115,6 @@ workflow {
     )
 }
 
-//
-// Get attribute from genome config file e.g. fasta
-//
-def getGenomeAttribute(attribute) {
-    if (params.genomes && params.genome && params.genomes.containsKey(params.genome)) {
-        if (params.genomes[ params.genome ].containsKey(attribute)) {
-            return params.genomes[ params.genome ][ attribute ]
-        }
-    }
-    return null
-}
```

Comment:
don't you want to keep this here?

Reply:
actually it works now from the function being included explicitly at top
`include { getGenomeAttribute      } from './subworkflows/local/utils_nfcore_methylseq_pipeline'`